### PR TITLE
Add Prometheus doc

### PR DESCRIPTION
**Pull request by ivyostosh**
*Issue #, if available:* https://github.com/aws/eks-anywhere-packages/issues/639

The rendered view of main changes can be accessed at https://prometheus-doc-deploy.d234yjlc0qbnov.amplifyapp.com/docs/tasks/packages/prometheus/ and https://prometheus-doc-deploy.d234yjlc0qbnov.amplifyapp.com/docs/reference/packagespec/prometheus/v2.39.1/

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/packagespec/_index.md [hunk 1]
```diff
@@ -10,10 +10,12 @@ description: >
...
-| [MetalLB]({{< relref "./metallb" >}}) | MetalLB is a virtual IP provider for services of type `LoadBalancer` supporting ARP and BGP. | [v0.12.1]({{< relref "./metallb/v0.12.1.md" >}}) | https://github.com/metallb/metallb/ |
-| [Emissary Ingress]({{< relref "./emissary" >}}) | Emissary Ingress is an open source `Ingress` supporting API Gateway + Layer 7 load balancer built on Envoy Proxy. | [v0.3.0]({{< relref "./emissary/v0.3.0.md" >}}) | https://github.com/emissary-ingress/emissary/ |
 | [ADOT]({{< relref "./adot" >}}) | ADOT Collector is an AWS distribution of the OpenTelemetry Collector, which provides a vendor-agnostic solution to receive, process and export telemetry data. | [v0.21.1]({{< relref "./adot/v0.21.1.md" >}}) | https://github.com/aws-observability/aws-otel-collector |
+| [Cert-manager]({{< relref "./cert-manager" >}}) | Cert-manager is a certificate manager for Kubernetes clusters. | [v1.9.1]({{< relref "./cert-manager/v1.9.1.md" >}}) | https://github.com/cert-manager/cert-manager |
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/packagespec/_index.md:14]: Nice idea to reorganized based on alphabetical order!

File: docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md [hunk 2]
```diff
@@ -0,0 +1,565 @@
...
+| server.name | Specifies prometheus-server container name | `"server"` |
+| server.persistentVolume.accessModes | Specifies prometheus-server data Persistent Volume access modes | `"ReadWriteOnce"` |
+| server.persistentVolume.enabled | Indicates if prometheus-server will create/use a Persistent Volume Claim | `true` |
+| server.persistentVolume.existingClaim | Specifies prometheus-server data Persistent Volume existing claim name. It requires `server.persistentVolume.enabled: true`. If defined, PVC must be created manually before volume will be bound | `""` |
+| server.persistentVolume.size | Specifies prometheus-server data Persistent Volume size | `"8Gi"` |
+| server.remoteRead | Specifies prometheus-server remote read configs. Refer to Prometheus docs [remote_read](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_read) for more details | `[]` |
+| server.remoteRead | Specifies prometheus-server remote write configs. Refer to Prometheus docs [remote_write](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) for more details | `[]` |
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md:59]:
```suggestion
| server.remoteWrite | Specifies prometheus-server remote write configs. Refer to Prometheus docs [remote_write](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) for more details | `[]` |
```
I'm guessing this is what you mean.

File: docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+| server.replicaCount | Specifies the replicaCount for prometheus-server deployment / statefulSet. **Note `server.statefulSet.enabled` should be set to `true` if `server.replicaCount` is greater than `1`**  | `1` |
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md:60]:
```suggestion
| server.replicaCount | Specifies the replicaCount for prometheus-server deployment / statefulSet. **Note:** `server.statefulSet.enabled` should be set to `true` if `server.replicaCount` is greater than `1`  | `1` |
```
Not required, but I like to just bold the "Note" part.

File: docs/content/en/docs/tasks/packages/prometheus/_index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+description: >
+  Install/upgrade/uninstall Prometheus
+---
+
+If you have not already done so, make sure your cluster meets the [package prerequisites.]({{< relref "../prereq" >}})
+Be sure to refer to the [troubleshooting guide]({{< relref "../../troubleshoot/packages" >}}) in the event of a problem.
+
+   {{% alert title="Important" color="warning" %}}
+   * Starting at `eksctl anywhere` version `v0.12.0`, packages on workload clusters are remotely managed by the management cluster.
+   * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be ran with `kubeconfig` pointing to the workload cluster.
```
Comment (chrisnegus) [inline docs/content/en/docs/tasks/packages/prometheus/_index.md:15]:
```suggestion
   * While following this guide to install packages on a workload cluster, please make sure the `kubeconfig` is pointing to the management cluster that was used to create the workload cluster. The only exception is the `kubectl create namespace` command below, which should be run with `kubeconfig` pointing to the workload cluster.
```

**Review by chrisnegus — CHANGES_REQUESTED** [review]
Reads really well.  I have a few suggestions.

File: docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md (hunk 2 above)
Comment (ivyostosh) [reply docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md]: Good catch! Updated.

File: docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md (hunk 3 above)
Comment (ivyostosh) [reply docs/content/en/docs/reference/packagespec/prometheus/v2.39.1.md]: Updated.

File: docs/content/en/docs/tasks/packages/prometheus/_index.md (hunk 4 above)
Comment (ivyostosh) [reply docs/content/en/docs/tasks/packages/prometheus/_index.md]: Thank you! I'm going to raise a separate PR to address this comment for other packages as well

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**